packages/apps/shopify-app-remix/package.json

```diff
@@ -16,12 +16,6 @@
   "module": "./dist/esm/server/index.mjs",
   "types": "./dist/ts/server/index.d.ts",
   "exports": {
-    ".": {
```

Comment:
Is this all we needed to change? 

I tested on a dev app, and it was now giving me an error when I tried to import without `/server`

Reply:
Yes, that is it! Unfortunately I couldn't leave a comment because this is a JSON file, but that's all we had to change.